lib/iex/lib/iex.ex

```diff
@@ -607,9 +607,10 @@ defmodule IEx do
     end
   end
 
-  def __break__!({:/, _, [call, arity]} = ast, stops, env) when is_integer(arity) do
+  def __break__!({:/, _, [call, arity]} = ast, stops, env)
+      when arity in 0..255 and is_integer(stops) and stops >= 0 do
     with {module, fun, []} <- Macro.decompose_call(call),
-         module when is_atom(module) <- Macro.expand(module, env) do
+         module when is_atom(module) and is_atom(fun) <- Macro.expand(module, env) do
```

Comment:
`is_atom(fun)` is already guaranteed by `decompose_call`, so I wouldn't check it again.